apps/src/code-studio/pd/workshop_dashboard/components/workshop_table_loader.jsx

```diff
@@ -96,7 +96,10 @@ export default class WorkshopTableLoader extends React.Component {
     if (this.state.loading) {
       return (
         // While reloading, preserve the height of the previous child component so the refresh is smoother.
-        <div style={{height: this.childHeight}}>
+        <div
+          style={{height: this.childHeight}}
+          data-testid={'enrolled-workshops-loader'}
```

Comment:
Added this `data-testid` to the spinner on the Workshops component to make sure this section is showing up for the new React Testing Library tests.